packages/radix-vue/src/HoverCard/HoverCardTrigger.vue

```diff
@@ -1,60 +1,35 @@
 <script lang="ts">
-export interface HoverCardTriggerProps extends PrimitiveProps {}
+export interface HoverCardTriggerProps extends PrimitiveProps { }
 </script>
 
 <script setup lang="ts">
-import { inject } from "vue";
-import {
-  PrimitiveButton,
-  usePrimitiveElement,
-  type PrimitiveProps,
-} from "@/Primitive";
+import { PopperAnchor } from "@/Popper";
+import { PrimitiveButton, type PrimitiveProps } from "@/Primitive";
+import { useHover } from "@/shared";
+import { inject, ref } from "vue";
 import {
   HOVER_CARD_INJECTION_KEY,
   type HoverCardProvideValue,
 } from "./HoverCardRoot.vue";
-import { PopperAnchor } from "@/Popper";
-
-import { useHoverDelay, useMouseleaveDelay } from "../shared";
 
 const injectedValue = inject<HoverCardProvideValue>(HOVER_CARD_INJECTION_KEY);
 
 const props = defineProps<HoverCardTriggerProps>();
-
-const { primitiveElement, currentElement } = usePrimitiveElement();
-
-async function handleMouseEnter(e: MouseEvent) {
-  const result = await useHoverDelay(
-    e,
-    currentElement.value!,
-    injectedValue?.openDelay
-  );
-  if (result) {
-    injectedValue?.showTooltip();
-  }
-}
-
-async function handleMouseleave(e: MouseEvent) {
-  injectedValue!.isHover = false;
-  const result = await useMouseleaveDelay(e, injectedValue?.closeDelay);
-  if (result && !injectedValue?.isHover) {
-    injectedValue?.hideTooltip();
-  }
-}
+const triggerElement = ref<HTMLElement>();
+
+const isHovered = useHover(triggerElement, {
```

Comment:
this `useHover` broke `HoverCard`. I suggest let's add back `useHoverDelay`, and we can remove that once we refactor HoverCard.